app/src/main/java/org/sil/storyproducer/controller/storylist/FilterToolbarFrag.kt

```diff
@@ -0,0 +1,55 @@
+package org.sil.storyproducer.controller.storylist
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.google.android.material.chip.Chip
+import com.google.android.material.chip.ChipGroup
+import org.sil.storyproducer.R
+import org.sil.storyproducer.model.Story
+
+
+class FilterToolbarFrag(private val storyPageFrag : StoryPageFragment): Fragment() {
```

Comment:
Could you add a comment about the scope of this class - is this what it does??
"FilterToolbarFrag implements all chip filters within a tab on the story list page"

Reply:
Done as well!